src/table/townname.h

```diff
@@ -2015,15 +2015,15 @@ static const char * const _name_romanian_real[] = {
 	"Sovata",
 	"Suceava",
 	"Sulina",
-	u8"T\u0103nd\u0103rei",
-	u8"T\u00e2rgoviste",
+	u8"T\u00e2rgovi\u0219te",
 	u8"T\u00e2rgu Jiu",
-	u8"T\u00e2rgu Mures",
+	u8"T\u00e2rgu Mure\u0219",
 	"Tecuci",
 	u8"Timi\u0219oara",
 	"Tulcea",
 	"Turda",
 	u8"Turnu M\u0103gurele",
+	u8"\u021a\u0103nd\u0103rei",
```

Comment:
You should not change the order of the names, even if that will make them sorted wrong in the list. If you change the order, existing saved games will suddenly have towns change name and confuse the player.